src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/IPAOpenDataServiceImpl.scala

```diff
@@ -70,7 +70,8 @@ object IPAOpenDataServiceImpl {
 
     response.records.flatMap { record =>
       for {
-        id             <- record(mapped(InstitutionsFields.id)).select[String]
+        id             <- record(mapped(InstitutionsFields.taxCode)).select[String]
```

Comment:
are we sure that tax code is unique across O, UO and AOO?

Reply:
@iwoak @beetlecrunch
As I understand, actually O, UO and AOO are not retrieved from actual origins.
When we will integrate those origins, the id will be built using the following concatenation:
CF#origin#origin_id

Reply:
yes @antonioT90 @galales: in this right moment we are accepting only main organitations, uniquely identified by CF.
In a future step we'll accept UO ad AOO and for these we'll found a way to create on id field a composite key.
cc @beetlecrunch

Reply:
@galales makes exactly the same remarks I did, in particular, to keep the data most compliant to the origin source.
The solution provided (composite key) was accepted previously, let us just keep in mind to be very careful adding new sources.

Reply:
@beetlecrunch it's better if I add a TODO in the lines where the AOO is put to None in order to remember to update id when AOO will be integrated?

Reply:
Yes, it's an easy change, but could be useful.